src/context/SubaccountContext/SubaccountContext.tsx

```diff
@@ -367,7 +368,7 @@ export function useSubaccount(requiredBalance: bigint | null, requiredActions =
   ]);
 }
 
-export function useSubaccountInsufficientFunds(requiredBalance: bigint | undefined | null) {
+export function useSubaccountInsufficientFunds(requiredBalance?: bigint | null) {
```

Comment:
my intention was to always provide argument to not forget pass balance where possible

but maybe that's a silly way, and it's fine to omit it

Reply:
Oh got it. Then I guess it may be better to force passing the required Balance even if it is default. In such case it should be manually received it outside of the function.